README.md

```diff
@@ -32,10 +32,9 @@ Under File -> Settings -> Editor -> Code Style import the google-styleguide (gea
 ## Loading initial test data
 1. In `mots` root directory run:
 ```
-mysql -uroot -p mots < src/main/resources/test_modules.sql
-mysql -uroot -p mots < src/main/resources/test_ivr_config.sql
+mysql -uroot -ppassword mots < src/main/resources/test_modules.sql
+mysql -uroot -ppassword mots < src/main/resources/test_ivr_config.sql
```

Comment:
let's remove it - password can be different on each env

Reply:
Ok